docs/Text.md

```diff
@@ -0,0 +1,185 @@
+# Adding Text
+
+There are several ways in fpdf to add text to a PDF document, each of which comes with its own special features and its own set of advantages and disadvantages. You will need to pick the right one for your specific task.
+
+| method | lines | markdown | HTML | positioning | details |
+| -- | :--: | :--: | :--: | :--: | -- |
+| [`.text()`](#text)  | one | no | no | fixed | Inserts a single-line text string with a precise location on the base line of the font.|
+| [`.cell()`](#cell)  | one | yes | no | yes | Inserts a single-line text string within the boundaries of a given box, optionally with background and border. |
+| [`.multi_cell()`](#multi_cell) | several | yes | no | yes | Inserts a multi-line text string within the boundaries of a given box, optionally with background and border. |
+| [`.write()`](#write) | several | no | no | auto | Inserts a multi-line text string within the boundaries of the page margins, starting at the current x/y location (typically the end of the last inserted text). |
+| [`.write_html()`](#write_html) | several | no | yes | auto | From [html.py](HTML.html). An extension to `.write()`, with additional parsing of basic HTML tags.
+
+## Typographical Limitations
+
+There are a few advanced typesetting features that fpdf doesn't currently support.
```

Comment:
Could you please add a newline before lists? Otherwise it won't get rendered properly in HTML / PDF

Reply:
Will do.